telos-backend/demo.js

```diff
@@ -0,0 +1,15 @@
+const TodoTask = require('./src/models/todo_task');
+
+TodoTask.find().countDocuments((err, count) => {
+  if (count === 0) {
+    console.log('Empty');
+    const entry = new TodoTask({
+      name: 'Lorem',
+      createdDate: new Date(2021, 1, 1),
```

Comment:
![savedToDb](https://user-images.githubusercontent.com/49116468/111897129-fbcf9900-8a82-11eb-98a8-2890fc044867.png)
When creating using this, its not quite what we expect I think. I think this is because the monthIndex starts from 0, see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date and search for `monthIndex` (its stupid, but they decided on that...)

Reply:
Just typing out the string '2021-01-01' should be all good, no need for date constructor

Reply:
Okay I will correct this now